prime-router/docs/universal-pipeline/receive.md

```diff
@@ -1,25 +1,50 @@
 # Universal Pipeline Receive Step
-`Receive` is the first step a message goes through when entering the Universal Pipeline (UP). Despite its name, this step
-actually occurs in the `ReportFunction.kt` file. 
+
+`Receive` is the first step a report goes through when entering the Universal Pipeline (UP). Despite its name, this step
+actually occurs in the `ReportFunction.kt` file.
 
 ## How it works
-There are two calls that will kick off this function, see the Swagger doc for specifics: 
-http://localhost:8080/swagger-ui.html. Currently, the UP can receive data in either HL7 or FHIR. We are planning to 
-support CSV in the future.
+
+There are two HTTP endpoints that will kick off this function: [/reports](../api/reports.yml)
+and [/waters](../api/waters-reports.yml). The UP can receive data in either HL7 or FHIR via these two endpoints.
+
+### Azure Function Access Key Security
+
+[Azure Function Access Key](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=python-v2%2Cisolated-process%2Cnodejs-v4%2Cfunctionsv2&pivots=programming-language-csharp#authorization-keys)
+security is supported by
+the `/reports` endpoint. Static and unchanging API Keys can be defined
+in the Azure environment for the Reports function and sent to the client. This method is less secure than FHIR Auth
+Security and thus the `/reports` endpoint should only be used over the `/waters` endpoint in special circumstances.
```

Comment:
Mostly just a question for myself, and maybe something that could be linked here. Are these special circumstances documented anywhere?

Reply:
"special circumstances" essentially just means the sender cannot support OAuth2.0 due to their technical capabilities.